app/build.gradle

```diff
@@ -20,9 +20,8 @@ android {
 
 dependencies {
     testImplementation 'junit:junit:4.12'
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'com.android.support:recyclerview-v7:28.0.0'
-    implementation 'com.loopj.android:android-async-http:1.4.9'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
+    implementation 'com.codepath.libraries:asynchttpclient:0.0.9'
     implementation 'com.github.bumptech.glide:glide:4.8.0'
-    implementation group: 'com.google.guava', name: 'guava', version: '18.0'
```

Comment:
this seems unused and I don't recall Guava being needed during implementation.

Reply:
Yeah I remember being confused by it..